python/neuroglancer/pipeline/scripts/ngmesh2obj.py

```diff
@@ -1,6 +1,9 @@
 #!/use/bin/python
 
 """
+WARNING: This script has been supersceded by the capabilities
```

Comment:
There should be a single way of doing something.

Reply:
Yea, the cloudvolume version is now MUCH better. I think you're right let's delete this.